# constants as actions , you can refer to them in your code
int32 RIGHT = 0
int32 UP = 1
int32 LEFT = 2
int32 DOWN = 3
# a plan is a sequence of actions
int32[] plan
---
# return whether plan could be executed successfully
bool success